clang/test/SemaSYCL/vec-type-hint.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -fsycl-is-device -sycl-std=2017 -fsyntax-only -DKERNEL= -verify=sycl-2017 %s
+// RUN: %clang_cc1 -fsycl-is-device -sycl-std=2020 -fsyntax-only -DKERNEL= -verify=sycl-2020 %s
+// RUN: %clang_cc1 -fsyntax-only -DKERNEL=kernel -verify=opencl -x cl %s
+
+// opencl-no-diagnostics
+// sycl-2017-no-diagnostics
+
+// __attribute__((vec_type_hint)) is deprecated in OpenCL mode without
```

Comment:
What is OpenCL mode here? Like when is it allowed? When we're not compiling SYCL code?

Reply:
I think this is a think-o on my part and the comment should be fixed. I mean to say that the GNU spelling is deprecated in SYCL 2020 mode without replacement, but is still allowed in SYCL 2017 and OpenCL mode.